server/src/pages/shot/editor.js

```diff
@@ -0,0 +1,150 @@
+const React = require("react");
+const sendEvent = require("../../browser-send-event.js");
+
+let pos;
+
+exports.Editor = class Editor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.draw = this.draw.bind(this);
+    this.setPosition = this.setPosition.bind(this);
+    this.state = {
+      tool: 'none',
+      color: '#000000',
+      size: '5'
+    };
+  }
+
+  render() {
+    let penState;
+    let highlighterState;
+    let canvasHeight = this.props.clip.image.dimensions.y;
+    let canvasWidth = this.props.clip.image.dimensions.x;
+    if (this.state.tool == 'highlighter') {
+      highlighterState = 'active';
+      penState = 'inactive';
+    } else if (this.state.tool == 'pen') {
+      highlighterState = 'inactive';
+      penState = 'active';
+    } else {
+      penState = 'inactive';
+      highlighterState = 'inactive';
+    }
+    return <div>
+      <div className="editor-header default-color-scheme">
+        <div className="shot-main-actions">
+          <a className={`button pen-button ${penState}`} id="pen" onClick={this.onClickPen.bind(this)} title="pen"></a>
```

Comment:
I'd generally prefer these to be buttons, since that's semantically more correct.